BrickController2/BrickController2.WinUI/Extensions/ControllerExtensions.cs

```diff
@@ -4,7 +4,8 @@ namespace BrickController2.Windows.Extensions;
 
 public static class ControllerExtensions
 {
-    public static string GetDeviceId(this Gamepad gamepad) =>
-        // kinda hack
-        gamepad.User.NonRoamableId;
+    // deviceId looks like "{wgi/nrid/]Xd\\h-M1mO]-il0l-4L\\-Gebf:^3->kBRhM-d4}\0"
+    // A unique ID that identifies the controller. As long as the controller is connected, the ID will never change.
+    // JK: RawGameController.FromGameController(gamepad) returns null on deleted gamepad
+    public static string GetUniquePersistentDeviceId(this Gamepad gamepad) => RawGameController.FromGameController(gamepad).NonRoamableId;
```

Comment:
I've found strange behavior if I remove the gamepad (USB one) and plug it again, readded `Gamepad` instance seems to cause an exception here:
![image](https://github.com/user-attachments/assets/9b033659-3500-4ecd-b58b-4e69c45f9ea8)